src/renderer/file-manager.ts

```diff
@@ -70,11 +70,16 @@ export class FileManager {
     for (const [name, value] of Object.entries(files)) {
       if (name === PACKAGE_NAME) {
         const { remoteLoader } = window.app;
-        const { dependencies, devDependencies } = JSON.parse(value);
-        const deps: Record<string, string> = {
-          ...dependencies,
-          ...devDependencies,
-        };
+
+        let deps: Record<string, string> = {};
+        try {
+          const { dependencies, devDependencies } = JSON.parse(value);
+          deps = { ...dependencies, ...devDependencies };
```

Comment:
```suggestion
        const deps: Record<string, string> = {};
        try {
          const { dependencies, devDependencies } = JSON.parse(value);
          Object.assign(deps, dependencies, devDependencies);
```

Would let `deps` remain `const`, which is nice.